src/util/locking/RedisLocker.ts

```diff
@@ -42,6 +36,12 @@ const PREFIX_LOCK = '__L__';
  * These scripts are used by Redis as a single new command.
  * Redis executes its operations in a single thread, as such, each such operation can be considered atomic.
  *
+ * The operation to un/lock will always resolve with either 1/OK/true if succeeded or 0/false if not succeeded.
```

Comment:
```suggestion
 * The operation to (un)lock will always resolve with either 1/OK/true if succeeded or 0/false if not succeeded.
```

Also similar comments as in the file locker about the references.